tests/conftest.py

```diff
@@ -1,8 +1,54 @@
+import time
+
 import pytest
 
 from veg2hab.io.cli import CLIInterface
 
+TIMED = "timed"
+SLOW = "slow"
+SLOW_OPTION = f"--run_{SLOW}"
```

Comment:
iets van je-hoeft-maar-op-1-plek-de-naam-te-veranderen vind ik altijd wel nice, of is dit wat overkill?

Reply:
Beetje overkill, maar is prima.

Reply:
nitpick: meestal gebruiken mensen `--run-slow` i.p.v. `--run_slow`

Reply:
verander ik :)